wiki/release/release-note-guide.md

```diff
@@ -0,0 +1,69 @@
+This guide explains everything about Pulsar release notes.
```

Comment:
```suggestion
# Release Note Guide

This guide explains everything about Pulsar release notes.
```

... and indent one level for all headings below.

Reply:
Good suggestion! I've incorporated and added a TOC

<img width="795" alt="image" src="https://user-images.githubusercontent.com/50226895/194993696-9f6bead4-3102-458b-9e23-ff192a5d5a8a.png">

Reply:
See my comment here https://github.com/apache/pulsar/pull/17470#discussion_r963277120 that on GitHub we don't need to manually maintain a TOC :)